crates/astria-sequencer/src/grpc/sequencer.rs

```diff
@@ -152,6 +157,43 @@ impl SequencerService for SequencerServer {
 
         Ok(Response::new(block))
     }
+
+    #[instrument(skip_all)]
+    async fn get_pending_nonce(
+        self: Arc<Self>,
+        request: Request<GetPendingNonceRequest>,
+    ) -> Result<Response<RawPendingNonce>, Status> {
+        use astria_core::primitive::v1::Address;
+
+        use crate::accounts::state_ext::StateReadExt as _;
+
+        let request = request.into_inner();
+        let Some(address) = request.address else {
+            return Err(Status::invalid_argument(
+                "required field address was not set",
+            ));
+        };
+
+        let address = Address::try_from_raw(&address)
+            .map_err(|e| Status::invalid_argument(format!("invalid address: {e}")))?;
```

Comment:
Please emit an event here, logging the entire error source chain.

Reply:
the error returned from `Address::try_from_raw` is `IncorrectAddressLength` so that's the only possible error, can add a trace log